contracts/LivepeerProtocol.sol

```diff
@@ -121,6 +121,22 @@ contract LivepeerProtocol {
     // rewardMultiplier[1] -> transcoder's cumulative stake for round
     mapping (address => mapping (uint256 => uint256[2])) public rewardMultiplierPerTranscoderAndRound;
 
+    // The various states a job can be in
+    enum JobStatus { Inactive, Unclaimed, Claimed, Verified }
+
+    // Represents a transcoding job
+    struct Job {
+        uint256 jobId;                     // Unique identifer for job
+        uint256 streamId;                  // Unique identifier for stream
```

Comment:
Let's remember to change `streamID` from `uint256` to it's more semantically proper type when we settle on how to represent `streamID` in the system. For now the concatenation of a node address + unique id in the same space might not be great since it's so long, but even that long I don't think it's a uint256.